tasks/main.yml

```diff
@@ -4,6 +4,58 @@
 - include: 'system_package_management.yml'
   tags: [ 'role::owncloud:pkg' ]
 
+- name: Create ownCloud enforce permissions script
+  template:
+    src: 'usr/local/lib/owncloud-enforce-permissions.j2'
+    dest: '{{ owncloud__enforce_permissions_script }}'
+    owner: 'root'
+    group: 'root'
+    mode: '0744'
+  tags: [ 'role::owncloud:base_install' ]
+  register: permissions_script
+  when: owncloud__permission_type == 'script'
+
+- name: Run enforce permissions script
+  command: '{{ owncloud__enforce_permissions_script }}'
+  when: permissions_script.changed and owncloud__permission_type == 'script'
+
+- name: StatOverride Set
+  command: "dpkg-statoverride --update --force --add {{ owncloud__user }} {{ owncloud__group }} {{ item.0.mode }} \"{{ item.1 }}\""
+  with_subelements:
+    - '{{ owncloud__statoverride }}'
+    - path
+  when: ( owncloud__deploy_state == 'present' and
+          owncloud__permission_type == 'dpkg' and
+          owncloud__user != '{{ ansible_local.nginx.user
+                              if (ansible_local|d() and ansible_local.nginx|d() and
+                                  ansible_local.nginx.user|d())
+                              else "www-data" }}' )
+
+- name: StatOverride UnSet
+  command: "dpkg-statoverride --update --force --remove \"{{ item.1 }}\""
```

Comment:
You can use the `quote` filter. Ref: https://docs.ansible.com/ansible/playbooks_filters.html#other-useful-filters